src/widgets/reflow.rs

```diff
@@ -6,6 +6,7 @@ use unicode_width::UnicodeWidthStr;
 use crate::{layout::Alignment, text::StyledGrapheme};
 
 const NBSP: &str = "\u{00a0}";
+const ZWSP: char = '\u{200b}';
```

Comment:
Is there a reason of this being a `char` whereas `NBSP` is `&str`?

Reply:
> Is there a reason of this being a `char` whereas `NBSP` is `&str`?

It's because ZWSP is compared with another char while NBSP is compared with the symbol itself. I am not familiar with Graphemes, maybe I could use symbol == ZWSP here?

Reply:
> I am not familiar with Graphemes, maybe I could use symbol == ZWSP here?

Well, it seems that's the case:

```
>> :dep  unicode-segmentation
>> use unicode_segmentation::UnicodeSegmentation;
>> "\u{200b}\u{200b} \u{00a0}\u{200b} \u{00a0}".graphemes(false).collect::<Vec<_>>()
["\u{200b}", "\u{200b}", " ", "\u{a0}", "\u{200b}", " ", "\u{a0}"]
```

I updated it to use &str like NBSP.